build.xml

```diff
@@ -22,18 +23,339 @@
+		    -->
+			<!--
+            <test name="org.scada_lts.dao.ViewHierarchyDaoTest"
+                haltonfailure="no">
+            </test>
+
+            <test name="org.scada_lts.service.ViewHierarchyServiceTest"
+                haltonfailure="yes">
+            </test>
+
+            <test name="org.scada_lts.service.ViewHierarchyServiceTestOnDB"
+                haltonfailure="yes">
+            </test>
+            -->
+			<!--
+            <test name="org.scada_lts.service.ViewHierarchyServiceMoveFolderTest"
+                haltonfailure="yes">
+            </test>
+
+            <test name="org.scada_lts.service.ViewHierarchyServiceMoveViewTest"
+                haltonfailure="yes">
+            </test>
+            -->
+			<test name="org.scada_lts.cached.DataPointsCacheComposeDataSourcesDataTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.cached.DataSourcePointsCacheMockitoTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataImage.PointValueCacheTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataImage.PointValueFacadeTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.web.dwr.DataPointDetailsDwrTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataImage.PointValueTimeTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataImage.AnnotatedPointValueTimeTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.dao.PointValueDAOTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.view.component.ViewComponentTestsSuite"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.dataSource.meta.ScriptExecutorTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.scripting.ContextualizedScriptRTTestsSuite"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.service.ScheduledExecuteInactiveEventServiceTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.vo.mailingList.MailingListInactive15MinuteIntervalTestsSuite"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.event.handlers.EmailToSmsHandlerRtTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.service.CommunicationChannelTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="com.serotonin.mango.rt.event.ScheduledExecuteInactiveEventRtTestsSuite"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.service.InactiveEventsProviderTest"
+				  haltonfailure="yes">
+			</test>
+
+			<test name="org.scada_lts.utils.UpdateValuePointPropertiesTest"
+				  haltonfailure="yes">
+			</test>
+		</junit>
+		<record action="stop" name="${junit.log.name}"/>
+
+	</target>
+
+	<target name="test-frontend-unit" description="Test Scada-LTS Frontend application">
```

Comment:
I would suggest: test-unit-frontend

Reply:
I used convention where you write <action>-<subproject>-<details> but your suggestion with <action>-<details>-<subproject> is also a good idea.

Reply:
If you think that is better okay, I understand there may be e.g. test-frontend-e2e, I think it makes sense.